src/webgpu/shader/execution/expression/binary/bitwise_shift.spec.ts

```diff
@@ -66,33 +62,34 @@ function is_valid_const_shift_right(e1: number, e1Type: string, e2: number) {
 
 // Returns all cases of shifting e1 left by [0,63]. If `is_const` is true, cases that are
 // invalid for const eval are not returned.
-function generate_shift_left_cases(e1: number, e1Type: string, is_const: boolean): Case[] {
-  const V = e1Type === 'i32' ? i32 : u32;
+function generateShiftLeftCases(e1: number, e1_type: ScalarType, is_const: boolean): Case[] {
+  const V = e1_type.create.bind(e1_type);
```

Comment:
Not too sure why this is needed. `el_type(e1)` should work just fine. Here and elsewhere